config/locales/zh-CN.yml

```diff
@@ -0,0 +1,229 @@
+---
+zh-CN:
+  thredded:
+    content_moderation_states:
+      content_blocked_notice: "被锁定"
+      content_blocked_notice_with_record_html: "%{moderator} 在%{time_ago}锁定"
+    email_notifier:
+      by_email: "电子邮箱"
+    emails:
+      message_notification:
+        html:
+          email_sent_reason_html: "我们给你发送这封邮件是因为 %{user} 在一个私人对话中提到你，“<a href=\"%{post_url}\">%{topic_title}</a>”。"
+          post_lead_html: "%{user} 在<a href=\"%{post_url}\">“%{topic_title}”</a>中发表了新消息："
+          unsubscribe_instructions_html: :thredded.emails.post_notification.html.unsubscribe_instructions_html
+        subject: "%{user} 在“%{topic_title}”中发表了新消息"
+        text:
+          email_sent_reason: |-
+            我们给你发送这封邮件是因为 %{user} 在一个私人对话中提到你，“%{topic_title}”。
+
+            请到以下浏览对话：
+            %{post_url}
+          post_lead: "%{user} 在“%{topic_title}”中发表了新消息："
+          unsubscribe_instructions: :thredded.emails.post_notification.text.unsubscribe_instructions
+      post_notification:
+        html:
+          email_sent_reason_html: "我们给你发送这封邮件是因为你关注了帖子： “<a href=\"%{post_url}\">%{topic_title}</a>”。"
+          post_lead_html: "%{user} 在<a href=\"%{post_url}\">“%{topic_title}”</a>中说到："
+          unsubscribe_instructions_html: "退订这些邮件, 请前往<a href=\"%{preferences_url}\">设置</a>。"
+        subject: "%{user} 在“%{topic_title}”中的新回复"
```

Comment:
This should not have the "%{user}", otherwise the email client will not group emails for the same topic together.
See https://github.com/thredded/thredded/commit/90c6f5fffd42ec1001c39dceb5ab5e875a71869d